chrononaut/versioned.py

```diff
@@ -154,38 +155,34 @@ def previous_version(self):
         )
         return chrononaut_snapshot_to_model(self, history_model) if history_model else None
 
-    def diff(self, from_timestamp, to_timestamp=None, include_hidden=False):
-        """Enumerate the changes from a prior model state (at ``from_timestamp``) to a later
-        model state or the current model's state (if ``to_timestamp`` is ``None``).
-
-        :param from_timestamp: A point in time for the model to diff from.
-        :param to_timestamp: A point in time to diff to or ``None`` for current version.
+    def diff(self, from_model, to=None, include_hidden=False):
```

Comment:
name the param `to_model` to match the function below?

Reply:
The original signature had this param named `to`, that's why it's named this way. I guess it comes down to compatibility vs consistency. I'd go with `to` since we're reinstating the old signature 🤷